src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -89,7 +74,11 @@ public Command parseCommand(String userInput) throws ParseException {
             return new ClearCommand();
 
         case FindCommand.COMMAND_WORD:
```

Comment:
Should find command be changed to not contain logic but an abstract class that the find commands inherit from?